From 8f0b44f039e2853ffd7e8c2b29f64b9e71e10e43 Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Sun, 28 Apr 2024 15:13:39 +0200
Subject: [PATCH] arm64: dts: imx8mp-hummingboard-pulse: use upstream rfkill
 properties

Upstream finally has accepted rfkill-gpio driver for use in device-tree.
Modify the properties to match current upstream choices.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 .../dts/freescale/imx8mp-hummingboard-pulse.dtsi     | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx8mp-hummingboard-pulse.dtsi b/arch/arm64/boot/dts/freescale/imx8mp-hummingboard-pulse.dtsi
index ba8f195350d9..ea4c76cf24c6 100644
--- a/arch/arm64/boot/dts/freescale/imx8mp-hummingboard-pulse.dtsi
+++ b/arch/arm64/boot/dts/freescale/imx8mp-hummingboard-pulse.dtsi
@@ -104,8 +104,8 @@ rfkill_m2_wdis {
 		compatible = "rfkill-gpio";
 		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_m2_rfkill>;
-		rfkill-name = "m2_wdis";
-		rfkill-type = <5>;
+		label = "m2_wdis";
+		radio-type = "wwan";
 		shutdown-gpio = <&gpio1 13 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
@@ -114,8 +114,8 @@ rfkill_m2_gps {
 		compatible = "rfkill-gpio";
 		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_m2_gps_rfkill>;
-		rfkill-name = "m2_gps";
-		rfkill-type = <6>;
+		label = "m2_gps";
+		radio-type = "gps";
 		shutdown-gpio = <&gpio1 7 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
@@ -124,8 +124,8 @@ rfkill_mpcie_wdis {
 		compatible = "rfkill-gpio";
 		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_mpcie_rfkill>;
-		rfkill-name = "mpcie_wdis";
-		rfkill-type = <5>;
+		label = "mpcie_wdis";
+		radio-type = "wwan";
 		reset-gpio = <&gpio1 1 GPIO_ACTIVE_HIGH>;
 		shutdown-gpio = <&gpio1 5 GPIO_ACTIVE_HIGH>;
 		status = "okay";
-- 
2.35.3

